package/upgrade/upgrade_manifests.sh

```diff
@@ -362,7 +362,7 @@ EOF
   pause_managed_chart harvester-crd "false"
 
   # XXX The eventual state of harvester would be "modified", not "ready"!
-  wait_managed_chart fleet-local harvester $REPO_HARVESTER_CHART_VERSION $pre_generation_harvester modified
+  wait_managed_chart fleet-local harvester $REPO_HARVESTER_CHART_VERSION $pre_generation_harvester ready
```

Comment:
Please help remove the above comment. Thanks.

Reply:
done